packages/big-bang-core/README.md

```diff
@@ -6,12 +6,11 @@ This package deploys [Big Bang Core](https://repo1.dso.mil/platform-one/big-bang
 
 ![helmreleases](./images/helmreleases.png)
 
-## Known Issues
-
-- Big Bang requires an AMD64 system to deploy as Iron Bank does not yet support ARM.  You will need to deploy to a cluster that is running AMD64.  Specifically, M1 Apple computers are not supported locally and you will need to provision a remote cluster to work with Big Bang currently.
-
 ## Instructions
 
+### macOS Users
```

Comment:
@willswire we actually just made some updates to where these instructions live to introduce a new Big Bang extension yesterday (see v0.25.0-rc1 when it comes out - its going through our release pipeline now) - can you move it as a `:::note :::` in the walkthrough's System Requirements instead?

https://docs.zarf.dev/docs/walkthroughs/big-bang
https://github.com/defenseunicorns/zarf/blob/main/docs/13-walkthroughs/5-big-bang.md

This is an example of how that looks when rendered in our docs:
![image](https://user-images.githubusercontent.com/3977569/222926996-c2a0e039-d211-49a7-a6b9-3c88dccd4999.png)

Reply:
with the latest update, I don't think my suggestions still apply! I'll refactor for the v0.25.0